src/webextension/widgets/input.css

```diff
@@ -17,12 +17,12 @@
   width: -moz-available;
   background-color: #f9f9fa;
   border: 1px solid rgba(12, 12, 13, 0.3);
-  border-radius: 2px;
+  border-radius: 3px;
```

Comment:
Where's this change coming from? It's not in the Photon spec, and the spec *looks* like the radius is 2px. @changecourse?

Reply:
@changecourse had reviewed that with me and opted for 3px. However, if we feel Photon is using 2px lets just revert to that. Will update.

Reply:
@nickbrandt Yeah, this is an area that the Photon design team needs to improve their specs, but it looked to me like 2px when comparing it to other widgets that actually have the border-radius specced.